test/integration/base_integration_test.cc

```diff
@@ -40,13 +46,13 @@ using ::testing::ReturnRef;
 
 BaseIntegrationTest::BaseIntegrationTest(const InstanceConstSharedPtrFn& upstream_address_fn,
                                          Network::Address::IpVersion version,
-                                         const std::string& config)
+                                         envoy::config::bootstrap::v3::Bootstrap bootstrap)
```

Comment:
pass bootstrap by const ref? The config_helper will make a copy and store it itself.